storybook/src/docs/components/StatusBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import './status-badge.css'
+import { Badge } from '@amsterdam/design-system-react/src'
+
+export const StatusBadge = ({
+  status = 'beta',
+  statusReason = '',
+  documentTop = true,
```

Comment:
This prop is never used, do we need it?

Reply:
My reasoning was to allow us to, occasionally, use the badge somewhere else than on the top of document.

Because it's `inline-flex` it could be used inline somewhere as well.

Vincent actually prefers to always have the component name on top

Reply:
Prop has been removed.